### DISCUSSION TITLE
Gibt es MouseOver noch?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#186] [Q&A]
Wir haben in einem Projekt die Anforderung erhalten, dass ein MouseOver für Objekte gewünscht wird.
Ist das in WebGIS5 möglich?

**Comment by gis-eni** [comment]
Nein, Mouse over haben wir aus dem Produkt rausgeschmissen.
Das hat nur Probleme gemacht. Nicht jeder arbeitet mit einer Maus usw.

**Reply by ue-black** [reply]
Was könnten wir als Alternative anbieten?

**Reply by gis-eni** [reply]
Abfragen?
